localstack-core/localstack/aws/handlers/validation.py

```diff
@@ -77,10 +83,11 @@ def __call__(self, chain: HandlerChain, context: RequestContext, response: Respo
 
         if path.startswith(f"{INTERNAL_RESOURCE_PATH}/") or path.startswith("/_aws/"):
             try:
-                self.openapi.validate_response(
-                    WerkzeugOpenAPIRequest(context.request),
-                    WerkzeugOpenAPIResponse(response),
-                )
+                for openapi in self.open_apis:
+                    openapi.validate_response(
```

Comment:
It would be great if we could cover the "multi-spec validation" in some kind of tests. Dynamic plugins for testing aren't straight forward, but a simple unit test would suffice and would be fairly simple I guess?